config/300-kaniko-clusterbuildtemplate.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: build.knative.dev/v1alpha1
```

Comment:
How will this integrate with @poy's Tekton work?

Reply:
This should be easy enough to port, Tekton already has a TaskRun template for Kaniko adapted from this one I stole from Cloud Build